extensions/sql-migration/src/api/azure.ts

```diff
@@ -620,6 +620,19 @@ export async function stopMigration(account: azdata.Account, subscription: Subsc
 	}
 }
 
+export async function deleteMigration(account: azdata.Account, subscription: Subscription, migrationId: string): Promise<void> {
```

Comment:
We could avoid these incidents by creating a small wrapper for the calls that new and old functions use. It will limit the code to defining the **path**, building the **requestBody** if necessary and returning the result of the call. It will also help reduce the amount of duplicate code. Probably an ADO by itself.
![image](https://user-images.githubusercontent.com/109680247/224056075-cd425e4c-29dc-4af4-99f5-ffa1ba293ceb.png)

Reply:
I like that idea for the other azure get by resource id api calls.  however, the deleteMigration is deletete call that returns void.

Reply:
refactoring the calls down would be a great next step